src/jit-dasm-pmi/jit-dasm-pmi.cs

```diff
@@ -482,6 +482,7 @@ void AddEnvironmentVariable(string varName, string varValue)
                     AddEnvironmentVariable("COMPlus_JitUnwindDump", "*");
                     AddEnvironmentVariable("COMPlus_JitEHDump", "*");
                     AddEnvironmentVariable("COMPlus_JitDiffableDasm", "1");
+                    AddEnvironmentVariable("COMPlus_ZapDisable", "1");
```

Comment:
For future-proofing we should also set `COMPlus_ReadyToRun=0` to disable R2R method loads too.